src/components/BulkActionsToolbar/BulkActionsToolbar.tsx

```diff
@@ -0,0 +1,98 @@
+import * as React from 'react';
+import { bem } from '../../utils';
+import { Button } from '../Buttons/Button';
+import { HandyTooltip } from './HandyTooltip';
+import styles from './BulkActionsToolbar.scss';
+
+export type ToggleState = 'all' | 'none';
+
+export type BulkActionsToolbarSelection = {
+    hasSelection: boolean;
+    label: SingleReactNode;
+    tooltip?: string;
+};
+
+export type BulkActionsToolbarToggle = {
+    selectAllLabel: string;
+    selectAllTooltip?: string;
+    selectNoneLabel: string;
+    selectNoneTooltip?: string;
```

Comment:
I'd prefer to keep tooltips as strings (it causes some issues with types and it will more time to solve them).
Will this work for you?
```ts
selectAllLabel: SingleReactNode;
selectAllTooltip?: string;
selectNoneLabel: SingleReactNode;
selectNoneTooltip?: string;
```

Reply:
I was concerned about `string` because we mostly use for translation `<Localized id={fieldset.fieldsetLabelI18nKey} />` component. However, we can use `getString` that returns `string`. It seems to have the same functionality and can be interchanged with `Localized`.